packages/flutter_test/test/widget_tester_test.dart

```diff
@@ -821,6 +822,78 @@ void main() {
       binding.postTest();
     });
   });
+
+  group('Accessibility announcements testing API', () {
```

Comment:
We should also test that clean-up logic from `postTest` works. Since the clean-up takes place as part of `testWidgets` after the test code is done, it can be done in `tearDown`, for example:

```dart
tearDown(() {
  expect(checkMockMessageHandler(SystemChannels.accessibility.name, null), isTrue);
});
```

Reply:
Done!